### DISCUSSION TITLE
New Public Repo responds with a 404?

### DISCUSSION THREAD

**Discussion by rivflux** [community/community#83503] [New to GitHub]
I have a new repo that is set to public, but for some reason if I try to view this while not logged in, using an incognito window, or using a different GitHub account it results in a "404 not found"

Why isn't this public?

http://github.com/rivflux/rivflux

**Comment by rivflux** [comment]
<img width="507" alt="Screenshot 2023-12-31 at 2 51 48 PM" src="https://github.com/community/community/assets/155249827/f9ca6abe-6de8-4e44-b189-7d012810bedc">

while logged in, it shows public...

<img width="452" alt="Screenshot 2023-12-31 at 2 52 27 PM" src="https://github.com/community/community/assets/155249827/1352cb77-54a9-4034-8506-d1179c0996ae">

incognito / private window shows 404

**Comment by queenofcorgis** [comment]
Hi there - I can confirm I am able to view this is and it is showing as public. Are you still experiencing this?

**Reply by rivflux** [reply]
Also 🥇 account name 🐶

**Comment by ravindudil5han** [comment]
Ensure that the repository is correctly set to public in the repository settings. Double-check the repository visibility, and make sure there are no restrictions or access limitations set. If the issue persists, you may want to clear your browser cache or try a different browser to rule out any local caching problems. If the problem continues, it's advisable to reach out to GitHub Support, providing specific details about the repository and the issue you're facing for further assistance.

**Comment by rivflux** [comment]
Ah sorry all. I should have closed this. 

I reached out to GitHub support and they had to unflag my account. All good now :] 

thank you!

**Reply by queenofcorgis** [reply]
No need to apologize @rivflux - glad Support got you sorted! And thanks for the username love 💟
